infrastructure/src/main/java/org/corfudb/infrastructure/BatchWriter.java

```diff
@@ -30,8 +30,13 @@
 public class BatchWriter<K, V> implements CacheWriter<K, V>, AutoCloseable {
 
     static final int BATCH_SIZE = 50;
+
+    final private boolean doSync;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)